src/scss/components/_gallery-item.scss

```diff
@@ -0,0 +1,62 @@
+/**
+  * @parent gaiden_css.components
+  * @stylesheet gaiden_css.components.gallery-item.css Gallery Item
+  *
+  * ## Services
+  * @demo demo/gaiden-css/components/gallery-item/demo.html
+**/
+.gallery-item {
+  width: 100%;
+  position: relative;
+  overflow: hidden;
+  border-radius: $borderRadius;
+  cursor: pointer;
+  transition: transform .2s linear;
+  will-change: transform;
+
+  &:hover {
+    transition: transform .2s linear;
+    transform: translateY(-7px) scale(1.03);
+  }
+
+  &__title {
+    width: 100%;
+    position: absolute;
+    bottom: 0;
+    border-radius: 0 0 $borderRadius $borderRadius;
+    overflow: hidden;
+
+    &::before {
+      width: 100%;
+      height: 100%;
+      position: absolute;
+      top: 0;
+      left: 0;
+      background-color: get-color((color: rastaGray, opacity: .5));
+      filter: blur(5px);
+      content: '';
+    }
+
+    div {
+      position: relative;
+      padding-top: 16px;
+      padding-bottom: 16px;
+      z-index: 2;
+      font-size: 14px;
+      font-weight: bold;
+      color: get-color(air);
+      text-transform: uppercase;
+      text-align: center;
+      background-color: get-color((color: punch, opacity: .8));
+    }
+  }
+
+  &__photo {
+    width: 100%;
+
+    img {
+      width: 100%;
+      float: left;
```

Comment:
The `float` is for recalculating the context, otherwise o gallery-item it looks like this:
![screen shot 2016-11-22 at 2 15 47 pm](https://cloud.githubusercontent.com/assets/4096456/20531785/f5445c9e-b0be-11e6-8a45-f302fe278621.png)

With the ` float` it look like this:
![screen shot 2016-11-22 at 2 22 30 pm](https://cloud.githubusercontent.com/assets/4096456/20531845/24de9a32-b0bf-11e6-9a5a-310f601fa6a1.png)